packages/tofu/src/codeSampleFromAstNode.js

```diff
@@ -1,5 +1,11 @@
 module.exports = { codeSampleFromAstNode }
 
+/**
+ *
+ * @param {import('type-fest').Merge<import('@babel/types').Node, {loc: import('@babel/types').SourceLocation}>} node
```

Comment:
Can we hide it somewhere? Put it in a types file and reference here by name or something? It's scary.

Reply:
I can create a `@typedef` for it, I suppose?